CMakeLists.txt

```diff
@@ -191,9 +191,9 @@ target_link_libraries(ni_grpc_device_server
    )
 
 if(WIN32)
-  target_link_libraries(ni_grpc_device_server Delayimp niScope niswitch nisync)
-  set_target_properties(ni_grpc_device_server PROPERTIES LINK_FLAGS "/DELAYLOAD:niScope.dll /DELAYLOAD:niswitch.dll /DELAYLOAD:nisync.dll")
-  set(ni_grpc_device_server CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS}  /DELAYLOAD:niScope.dll /DELAYLOAD:niswitch.dll /DELAYLOAD:nisync.dll")
+  target_link_libraries(ni_grpc_device_server Delayimp niScope niswitch nisync nitclk)
```

Comment:
I think this is supposed to be name of .lib file added. So it should be niTClk I guess since name of lib file is niTClk.lib. @ShantanuShinde  or @ni-pkhare can you guys confirm once?

Reply:
Yes and the name of the dll should be confirmed once by extracting this lib file(easiest way is to use 7Zip and check)

Reply:
When I tried, it seems to be `niTClk_64.dll`. @vijayinvites  you can confirm it once and update that name in the cmakelists.txt in 2 places I guess. @Akshata440 , only 2 places know